src/swtpm-wrapper

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+# Clean up tmpdir when terminated
+tpm_dir="$1"
+shift
+setpriv --pdeathsig SIGTERM swtpm socket \
+        --terminate \
+        --tpmstate dir="${tpm_dir}" --tpm2 \
+        --ctrl type=unixio,path="${tpm_dir}/swtpm-sock" &
+trap 'rm -rf ${tpm_dir}' EXIT
+# Wait around to remove the tmpdir
+setpriv --pdeathsig SIGTERM sleep infinity
```

Comment:
Hmm, seems like we can just `wait` here?

Reply:
Done!